[BD-6195] contribute component - bpk page indicator (#2604)
Skyscanner/backpack: UNRELEASED.md

**Added**

- bpk-component-page-indicator

>   - The page indicator is a new component that will be used to display multiple swappable images.

Reviewer: Sorry, you have to update it again since this component doesn't have swappable functionality

Author: Have updated a new instrument, hope to be reviewed again! Thank you!

- bpk-component-accordion:
    - Remove redundant `aria-labeledby` tags in accordion item

**Added**

- bpk-component-page-indicator

>   - The page indicator is a new component that will ensure the consistency of the image gallery display indicator.

Reviewer: ```suggestion
  - The page indicator is a new component that will be used to display multiple swappable images.
```

Author: Have updated, thank you!
